python/lsst/cp/pipe/ptc.py

```diff
@@ -85,27 +84,17 @@ class MeasurePhotonTransferCurveTaskConfig(pexConfig.Config):
         doc="Degree of polynomial to fit the PTC, when 'ptcFitType'=POLYNOMIAL.",
         default=3,
     )
+    linearity = pexConfig.ConfigurableField(
```

Comment:
I see that this is used below when calling this new linearity subtask, but I don't quite understand why this is a configuration field. (why is it here at this point?). I'm probably missing something.

Reply:
By making this a `ConfigurableField`, the target task can be instantiated as a subtask (L334).  This allows the PTC code to call the subtask's run method directly (L448).  This moves the linearity configuration to `config.linearity.polynomialOrder`and similar fields, instead of being directly part of the `MeasurePhotonTransferCurveTaskConfig`.